tools/tsp-client/src/index.ts

```diff
@@ -218,6 +219,18 @@ const parser = yargs(hideBin(process.argv))
       await sortSwaggerCommand(argv);
     },
   )
+  .command(
+    "compare",
+    "Compare two Swaggers for functional equivalency. This is typically used to compare a source Swagger with a TypeSpec-generated Swagger to ensure that the TypeSpec project is functionally equivalent to the source Swagger.",
+    (yargs: any) => {
+      return yargs.help(false);
```

Comment:
Well, it will do whatever the underlying script does, which is, correct, it doesn't give an error if they pass an unsupported flag. But that's something I can fix within that tool.

Reply:
https://github.com/tjprescott/openapi-diff/issues/14

Reply:
Also does the tool accept positional params? I suppose that will have to be considered in the logic of what it accepts or not. Also, we should add some info to the tsp-client readme about this new command and mention any flags folks should pass in

Reply:
I'll update the README. It doesn't accept positional params. You have to specify --lhs and --rhs (or have them set in dotenv or environment variables).